### PR TITLE
B19976

### PR DESCRIPTION
**Pull request by CoryKleinjanCACI**
## Summary

This task adds the ability for the prime to add a standaloneCrate option to the Domestic Crating item. Its functionality is nearly identical to a standard crating item, with the caveat that when the TOO goes to approve the item, the highest dollar amount that can be approved is 1000$. The TOO approval is part of a follow-on task and is not completed here.

Code removal is from the previous solution for 19976 that added an additional service item. Code had been merged into integration, so it had to be removed here.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

Prime UI:
1. Access the Office application
2. Log in to the Prime Simulator
3. Select 'Add a Service Item'
4. Select to add a Domestic Crate from the Service Item Type dropdown
5. Select domestic crating from the next dropdown.
6. Check the Standalone Crate option
7. Complete the form
8. Submit the form
9. Verify that the Service Item was added with a standalone tag in the DB.

Swagger:
PayLoad: {"item":{"length":4,"width":4,"height":4},"crate":{"length":5,"width":5,"height":5},"moveTaskOrderID":"1035d1f9-fe75-498f-98b3-21788d3084c3","mtoShipmentID":"5b600a58-944b-49ed-ba99-390886ab3d3e","modelType":"MTOServiceItemDomesticCrating","standaloneCrate":true,"reason":"i want to","description":"moped","reServiceCode":"DCRT"}

URL:http://officelocal:3000/swagger-ui/prime.html

Endpoint: mto-service-items

1. Update the supplied payload with an existing moveTaskOrderID and mtoShipmentID from your environment that is available to the prime.
2. Submit the payload to the provided endpoint.
3. Verify in the DB that the service item was added with a standalone tag.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

## Screenshots
Screenshot 2024-05-21 at 7 47 01 PM

Screenshot 2024-05-21 at 7 44 38 PM

### CODE REVIEW

**Comment by r-mettler** [comment]
Commit [040ea4194531c6d08b314ce0ba2835357a1d0ac0](https://github.com/transcom/mymove/pull/12747/commits/040ea4194531c6d08b314ce0ba2835357a1d0ac0z) is in INT but not in main because 19976 had an initial solution merged into INT but not yet into main and the change was replaced with a new solution and removed from INT with this commit.

File: pkg/handlers/primeapi/payloads/payload_to_model.go [hunk 1]
```diff
@@ -530,6 +531,7 @@ func MTOServiceItemModel(mtoServiceItem primemessages.MTOServiceItem) (*models.M
 				Width:  unit.ThousandthInches(*domesticCrating.Crate.Width),
 			},
 		}
+		model.StandaloneCrate = domesticCrating.StandaloneCrate
```
Comment (r-mettler) [inline pkg/handlers/primeapi/payloads/payload_to_model.go:534]: Potential merge duplicate here. Its already set on line 519

File: pkg/models/mto_service_items.go [hunk 2]
```diff
@@ -65,7 +65,8 @@ type MTOServiceItem struct {
 	CustomerExpense                   bool                           `db:"customer_expense"`
 	CustomerExpenseReason             *string                        `db:"customer_expense_reason"`
 	SITDeliveryMiles                  *int                           `db:"sit_delivery_miles"`
-	PricingEstimate                   *int                           `db:"pricing_estimate"`
+    PricingEstimate                   *int                           `db:"pricing_estimate"`
```
Comment (r-mettler) [inline pkg/models/mto_service_items.go]: Nit: The alignment of PricingEsitmate line somehow got off

**Review by r-mettler — APPROVED** [review]
Changes look good

**Review by antgmann — APPROVED** [review]

**Pull request merged**